ethereum/test/TablelandRigs.ts

```diff
@@ -532,41 +532,39 @@ describe("Rigs", function () {
       const table1 = "table1";
       const table2 = "table2";
       const table3 = "table3";
-      const uri = getURITemplate(tablelandHost, table1, table2, table3, false);
-      const part0 =
-        tablelandHost +
-        "/query?extract=true&unwrap=true&s=" +
-        encodeURIComponent(
-          `select json_object('name','Rig #'||rig_id,'external_url','https://garage.tableland.xyz/rigs/'||rig_id,'image','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_name,'image_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_full_alpha_name,'image_medium','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_name,'image_medium_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_medium_alpha_name,'thumb','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_name,'thumb_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_alpha_name,'animation_url',animation_base_url||rig_id||'.html','attributes',json_array(json_object('trait_type','status','value','pre-reveal'))) from table1 join table2 where rig_id=`
-        );
-      expect(uri[0]).to.equal(part0);
-      const part1 = encodeURIComponent(" group by rig_id;");
-      expect(uri[1]).to.equal(part1);
+      const uriTemplate = await getURITemplate(
+        tablelandHost,
+        table1,
+        table2,
+        table3,
+        false
+      );
+      const uri = uriTemplate.join("1");
+      const uriParts = uri.split("statement=");
+      expect(uriParts.length).to.equal(2);
+      const statement = decodeURIComponent(uriParts[1]);
```

Comment:
```suggestion
      const uri = new URL(uriTemplate.join("1"));
      const statement = uri.searchParams.get("statement");
      // normalizing the statement checks that it is valid SQL, will throw
      // an error if it isn't
```
This is a super small nitpick but we could use the `URL` class here, might be a bit nicer? And a comment that mentions that calling `normalize` checks that the statement is valid SQL might make sense since we don't have an explicit assert for that?